lib/dokumi/tool/android/lint.rb

```diff
@@ -0,0 +1,27 @@
+module Dokumi
+  module Tool
+    class Android
+      class Lint
+          LINT_REPORT_FILE = "build/outputs/lint-results-debug.xml"
+        
+        class << self
+          def parse_report(target_project)
+            report_path = Support.make_pathname(target_project).join(LINT_REPORT_FILE)
+            File.open(report_path) do |file|
+              report = Nokogiri::XML(file)
+
+              report.xpath("//issue").map do |issue|
+                {
```

Comment:
According to our styleguide, you should not break lines here.

https://github.com/cookpad/styleguide/blob/master/ruby.en.md#hashes